jobs/decorators/audit_decorators.py

```diff
@@ -67,61 +67,62 @@ def __init__(self, *model_classes):
         """
         if not model_classes:
             raise ValueError('Must target at least one model')
-        self._model_classes = set()
+        self._targeted_models = set()
```

Comment:
Codebase convention is to use "models" for model instances. If you want to refer to model classes, better to say "model classes" instead.